packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeDetailsItem/MT5TradeDetailsItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, useRef } from 'react';
+import { useHover } from 'usehooks-ts';
+import { qtMerge, Text, useBreakpoint } from '@deriv/quill-design';
+import { Clipboard, Tooltip } from '../../../../../components/Base';
```

Comment:
The `Clipboard` component is being added in [this PR](https://github.com/binary-com/deriv-app/pull/12427)
once the said PR is merged the import error will resolve